### PR TITLE
Add readme file and data base setup

### PR DESCRIPTION
**Pull request by Divinity-dev**
In this phase of the project, I have accomplished the following;
1. created a database for a vet store which includes id, name, date_of-birth, neutered, escape_attempts, and weight.
2. I have succeeded in filling the various data fields.
3. I have also implemented a search filter which includes;
 Find  animals whose name ends with "mon"
image1
Find animals born between 2016 and 2019.
image2
Find animals that are neutered and have less than 3 escape attempts.
image3
Find animals named either "Agumon" or "Pikachu".
image4

List name and escape attempts of animals that weigh more than 10.5kg
image5

Find all animals that are neutered.
image6
Find all animals not named Gabumon.
image7

Find all animals with a weight between 10.4kg and 17.3kg 
image8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,144 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon
+
+- [Live Demo Link](https://google.com)
```
Comment (alexander16108) [inline README.md:63]: remove the live demo link because it is irrelevant and serves no purpose 👍🏾

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Gabumon', '2018-11-15', 2, true, 8);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Pikachu', '2021-01-07', 1, false, 15.04);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Devimon', '2017-05-12', 5, true, 11);
```
Comment (alexander16108) [inline data.sql:8]: as seen in the queries the weight entry should be a float number and not a whole number as u represented above 👍🏾 please change it and put in the right entries

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Divinity-dev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [ ] as seen in the screenshot your entries are in duplicates please fix this and submit the new and updated screenshots

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Divinity-dev 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Good job implementing the requested changes.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
Thank you 

Sent from Yahoo Mail on Android 
 
  On Tue, Apr 4, 2023 at 10:04 AM, Kamwa Emmanuelle ***@***.***> wrote:   


@emmanuellekamwa approved this pull request.

STATUS: APPROVED 🍾 ✔ 🎉
Hello @Divinity-dev 👋
Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉
 
Highlights
   
   - You documented your work professionally. ✔
   - Good job implementing the requested changes.
   - Your linters are passing all the checks.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**